instancio-tests/feature-tests/src/test/java/org/instancio/test/features/feed/FeedFormatCsvTest.java

```diff
@@ -52,13 +53,7 @@ void csvWithTrimmedValue() {
 
         final String data = createCsv(commentPrefix, separator);
 
-        final Feed result = Instancio.ofFeed(Feed.class)
```

Comment:
Could you please revert changes to the existing tests?

Reply:
I believe you're referring to the static imports. If so, I reverted them